sdks/java/extensions/sql/zetasql/src/main/java/org/apache/beam/sdk/extensions/sql/zetasql/ZetaSQLPlannerImpl.java

```diff
@@ -171,6 +178,18 @@ public RelRoot rel(String sql, QueryParameters params) {
                 SqlAnalyzer.USER_DEFINED_FUNCTIONS,
                 String.join(".", createFunctionStmt.getNamePath()));
```

Comment:
I suppose I should have used getNamePath instead of getFullName...

Reply:
Ah you mean use a List<> than joined name? I think they are equivalent but have a common way to use function name is a good idea.

Reply:
Yeah, I will do that as a follow-up to avoid merge conflicts.